### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by alexawesomecode**
This is a pull request that applies the requirements needed for the apple-clone-project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            <div class="news-row-2">
+                <img src="img/promo_iphone5c.jpg" alt="iphone5c promo">
+
+            </div>
+
+            <div class="news-row-3">
+                <img src="img/promo_30_years.jpg" alt="30 years promo">
+
+            </div>
+
```
Comment (uche-inyama) [inline index.html]: The images should be wrapped in an **```<a>```** tag. The images are link to another articles or `#`

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi alex! nice job with the improvisation :slightly_smiling_face: 

But there are some updates we need to make.

- [ ] Your Html document did not pass a validation test, kindly use this [link](https://validator.w3.org), to check for the validation error.

- [ ] Using sematic html tags; wrap the main content of your html document in a **```<main> tag```** and use **```<nav> tag```** for the navigation section.

- [ ] Implement the hover effect on your on your navigation links.

## After implementing these changes submit another form a code review :slightly_smiling_face:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Apple</title>
+</head>
+
+<main>
```
Comment (samgaco) [inline index.html:12]: :+1:

File: style.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  border-right: 2px solid gray;
+  padding: 8px;
+  color: white;
+  font-weight: 200;
+  font-family: sans-serif, Arial;
+  font-size: 14px;
+  cursor: pointer;
+}
+
+.nav-bar li:hover {
```
Comment (samgaco) [inline style.css:55]: :100:

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi Alex :high_brightness: 

Everything required has been implemented, you only have left the following point:

- [ ] **[REQUIERED]** You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.

Please :pray: work on that and submit back for another code review once you are done.

Regards! :muscle: 

-------------------
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Comment by alexawesomecode** [comment]
done! 
fixed the mispelling
thanks,

alex

**Comment by alexawesomecode** [comment]
hello

i have done the checking a couple of times and it doesn't give me any error,

i am using this: https://validator.w3.org

**Review by uche-inyama — APPROVED** [review]
## Hi alex! 

Everything looks good now.

Go ahead and merge with the master branch :slightly_smiling_face: 

#### ```Have a great day``` :slightly_smiling_face:

**Pull request merged**